src/lib/network_pool/pool_sink.ml

```diff
@@ -158,7 +158,7 @@ module Base
               | None ->
                   let diff = Envelope.Incoming.data env' in
                   [%log error]
```

Comment:
```suggestion
                  [%log trace]
```

maybe, if it's not that big a deal?

Reply:
Maybe, but this seemed easier than making that concrete decision.